enterprise/backend/test/metabase_enterprise/sandbox/api/gtap_test.clj

```diff
@@ -26,6 +26,7 @@
    :card_id              true
    :table_id             true
    :group_id             true
+   :permission_id        false
```

Comment:
Letting this be `false` by default for now since most of these tests aren't mocking corresponding entries in the `permissions` table for the sandboxes that they create.